### ISSUE TITLE
Server: exit in case of db connection failure

### ISSUE THREAD

**Issue by oguzhanunlu** [snowplow/iglu#400]
Iglu Server hangs on the air in case of invalid db connection credentials, which makes the server look like running when actually it doesn't.

Server needs to fail with a user friendly error message instead of error's stack trace.

**Comment by chuwy** [comment]
Makes sense. Also it would be nice to exit immediately, not waiting for first HTTP request.

**Comment by oguzhanunlu** [comment]
Server checks if `apikey` and `schemas` tables exist in the beginning. It'll exit there if db credentials are invalid.
